services/integration/src/models/constitution/processes.model.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
```

Comment:
This file should be named as process.js instead of processes.js

Reply:
i changed the name´s file